war/src/main/less/abstracts/theme.less

```diff
@@ -189,6 +189,12 @@
   --link-text-decoration--hover: underline;
   --link-text-decoration--active: underline;
   --link-font-weight: 600;
+
+  // Tooltips
+  --tooltip-background-color: white;
```

Comment:
would it work to default to the background color and text color?

We can override this in dark theme / other themes but they may work out of the box if we stick to the existing palette, I did expect the table styled tooltip to work with the themes out of the box but it didn't

![image](https://user-images.githubusercontent.com/21194782/135583999-b43e4446-72a7-4d18-8c93-4b5f6d0079a3.png)


![image](https://user-images.githubusercontent.com/21194782/135584032-998bfaf0-4359-4e5e-af8d-d09263315c79.png)

![image](https://user-images.githubusercontent.com/21194782/135584059-9d6a9c6a-2da9-4e8b-a15d-2f11e9666317.png)

Reply:
I've update the variables to follow `--background` and `--text-color` so that it works better across themes :) I'll look into opening a PR for the dark theme repo to adjust the `--tooltip-background-color` as right now its a little too dark and blends too easily with the background.

<img width="130" alt="image" src="https://user-images.githubusercontent.com/43062514/136095157-343d5bbe-5f0a-4cc8-b7b1-ede4d8cd2d29.png">

Reply:
Thanks, looks much better than before, this also fixes https://github.com/jenkinsci/dark-theme-plugin/issues/83 by the looks of it